### ISSUE TITLE
Slim down package size

### ISSUE THREAD

**Issue by HIT2022** [TxnLab/use-wallet#8]
Just a suggestion, but for performance purposes (reducing bundle size) would be great if package had a lower footprint. Showing 206k 👍

Thanks for releasing this package! I'm new to the Algo eco, and although the tooling/documentation appears to be thorough, I was getting concerned that I wasn't able to find an existing JS package that abstracts connecting & transacting via the various wallet softwares behind a common interface.

**Comment by gabrielkuettel** [comment]
Fair point! I combed through the code and was able to reduce the package size down to 110k. Published to NPM as 0.1.16.

The remaining culprit in the package size is a base64 encoded audio file that we play on iOS devices when signing transactions. This is an awkward hack to keep the Wallet Connect web socket alive when iOS goes into background mode... we'll remove that as soon as there's a better solution.
